### ISSUE TITLE
Send 2 Unreal - Runtime Error Image is not Editable

### ISSUE THREAD

**Issue by Dprasni** [poly-hammer/BlenderTools#86]
Bug

`cannot import name 'make_remote' from 'send2ue.dependencies.rpc.factory' (C:\Users\Dragosh\AppData\Roaming\Blender Foundation\Blender\4.2\scripts\addons\send2ue\dependencies\rpc\factory.py)
`

When trying to turn on the addon.

Blender 4.2.1 - MS Store version

**Comment by jack-yao91** [comment]
This sounds like an issue with python modules not reloading after installing a new addon code. Can you:

1. Deactivate and remove send2ue addon
2. Shutdown blender
3. Start blender
4. install send2ue addon and activate it

Let me know if that works thanks

**Comment by Dprasni** [comment]
Yeah, seems it worked now for some reason after doing it on and off like 3 times, weird.

However now a different issue arose.
After trying to push assets I get this:
`Python: RuntimeError: Error: Image is not editable`

**Comment by JoshQuake** [comment]
TIL blender is on the MS store. I will see if I can replicate the issue

**Comment by Dprasni** [comment]
[light_collection_test_1.zip](https://github.com/user-attachments/files/17267102/light_collection_test_1.zip)
Here

**Comment by JoshQuake** [comment]
Hmm okay I'm not getting that error still. Can you please try again and screenshot/paste contents of the console?
![image](https://github.com/user-attachments/assets/9d7d79e6-d745-4751-a84f-9bd4f6f53f37)

I wonder if it's a file permissions issue?

**Comment by Dprasni** [comment]
![image](https://github.com/user-attachments/assets/5ad5b5e8-b056-4bed-b6da-2e2f5ce7cce2)

Just tried opening my "main" file, which is 400 megs big, pushing assets from that gives me reliably the said error.
Here's a drive link to it.
https://drive.google.com/file/d/11RlWohtDnvO5f9Z7tOC2CLV97Op9WadW/view?usp=drive_link

**Comment by JoshQuake** [comment]
Hmm ye still not getting any issues aside from a couple empty objects. I'm thinking it could be happening when trying to load Blender's FBX module. Since you are using the Microsoft Store version, I'm thinking there are permission issues.

My current suggestions are to find where the MS store version gets installed and change the permissions to allow full access, or you can try downloading Blender from blender.org (or Steam if you still want the easier updates)

**Comment by jack-yao91** [comment]
I think this is related to the unpacking of textures to disk so that FBX can read them correctly on export. 
https://github.com/poly-hammer/BlenderTools/blob/eb00c749664ee794b4c6918c05884157926f30c9/src/addons/send2ue/core/utilities.py#L1625

My guess is since this is only happening with this certain file, you might just try unpacking and see if that gives the same permission error about the image texture. 
![image](https://github.com/user-attachments/assets/82380ab5-1bc7-4901-ad01-c35cab6618ad)

I would check the folder permission where the .blend file is located

**Comment by Dprasni** [comment]
Tried unpacking, stull the same error occurs

**Comment by JoshQuake** [comment]
Could be permissions issue with either textures location, blender install, or project folder

**Comment by Dprasni** [comment]
Well i use a Blender UWA from the MS store because i really dislike the normal install version. The UWA apps are sandboxed and heavily walled up from the rest of the system and some other access restriction, so it could be that. This might be something a Windows developer could know more about.

**Comment by JoshQuake** [comment]
As this is caused by something out of our control (UWA being sandboxed disallowing access to Blender modules) I will be closing out this issue. Sorry we couldn't do anything for you here.
